androidApp/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```

Comment:
Why?

Reply:
This should indeed not be part of the boilerplate IMO

Reply:
Pretty sure it was crashing without it. I don't remember the reason though

Reply:
Fixed, it was a problem with lint. Picasso can post notifications if you, for example, download an image. If you don't use that feature, you don't need to specify the permission, but you must tell lint to ignore it in the build.gradle file